Jenkinsfile

```diff
@@ -701,60 +563,39 @@ pipeline {
-            withCredentials([file(credentialsId: 'Azure_Secrets_File', variable: 'Azure_Secrets_File')]) {
-              build job: 'tool-turn-on-slaves', parameters: [string(name: 'RoleNameAndRGname', value: 'azure-slave-1@kernel_pipeline')], wait: false
-              cleanWs()
-              unstash 'azure-linux-automation'
-              unstash "${env.KERNEL_ARTIFACTS_PATH}"
-              unstash 'kernel_version_ini'
-              unstash 'azure.env'
-              script {
-                  env.ARM_IMAGE_NAME = readFile 'ARM_IMAGE_NAME.azure.env'
-                  env.ARM_OSVHD_NAME = readFile 'ARM_OSVHD_NAME.azure.env'
-                  env.KERNEL_PACKAGE_NAME = readFile 'KERNEL_PACKAGE_NAME.azure.env'
-              }
-              RunPowershellCommand('cat scripts/package_building/kernel_versions.ini')
-              RunPowershellCommand(".\\RunAzureTests.ps1" +
-              " -ArchiveLogDirectory 'Z:\\Logs_Azure'" +
-              " -customKernel 'localfile:${KERNEL_PACKAGE_NAME}'" +
-              " -DistroIdentifier '${BUILD_NAME}${BUILD_NUMBER}'" +
-              " -ARMImageName '${ARM_IMAGE_NAME}'" +
-              " -testLocation 'centralus'" +
-              " -testCycle 'PERF-FIO'" +
-              " -OverrideVMSize 'Standard_DS14_v2'" +
-              " -ResultDBTable 'Perf_Storage_Azure_MsftKernel'" +
-              " -ResultDBTestTag 'FIO-12DISKS'" +
-              " -StorageAccount 'NewStorage_Premium'" +
-              " -ExitWithZero"
-              )
-              junit "report\\*-junit.xml"
-              RunPowershellCommand(".\\Extras\\AnalyseAllResults.ps1")
-            }
+            println("TBD")
           }
         }
+
         stage('validation_perf_hyperv') {
           when {
             beforeAgent true
-            expression { params.DISTRO_VERSION.toLowerCase().contains('centos') }
             expression { params.ENABLED_STAGES.contains('validation_perf_hyperv') }
+            expression { params.DISTRO_VERSION.toLowerCase().contains('ubuntu') }
           }
           agent {
             node {
-              label 'jenkins-meta-slave'
+              label "net_perf"
             }
           }
           steps {
-              dir('kernel_version' + env.BUILD_NUMBER + env.BRANCH_NAME) {
-                unstash 'kernel_version_ini'
-              }
-              script {
-                    iniPath = "${env:WORKSPACE}/kernel_version${env:BUILD_NUMBER}${env:BRANCH_NAME}/scripts/package_building/kernel_versions.ini"
-                    kernelPath = sh (script: "crudini --get ${iniPath} KERNEL_BUILT folder", returnStdout: true)
-                    build job: "performance-jobs/Pipeline_Performance", parameters: [
-                        string(name: 'KERNEL', value: "${kernelPath}")], wait: false;
-              }
-              echo "Triggered Local Hyper-V Performance tests."
-            
+            println("TBD")
+          }
+        }
+
+        stage('validation_sriov_hyperv') {
```

Comment:
this needs to go after the validation stage, then followed by performance

Reply:
will be done when impplemented